aws/lambda.mk

```diff
@@ -0,0 +1,20 @@
+# Macroses
```

Comment:
Why do we need this? Can we use `sls invoke` and `sls invoke local`? Sounds less intrusive.

Reply:
fixed.